scss/_maps.scss

```diff
@@ -172,3 +172,8 @@ $utilities-links-underline: map-loop($utilities-colors, rgba-css-var, "$key", "l
 $negative-spacers: if($enable-negative-margins, negativify-map($spacers), null) !default;
 
 $gutters: $spacers !default;
+
+// scss-docs-start navbar-breakpoints
+$navbar-breakpoints: $grid-breakpoints !default;
+// scss-docs-end navbar-breakpoints
```

Comment:
Hello @louismaximepiton. The extra sass var is used to keep the two breakpoints separate. If we alter the `$grid-breakpoints` map before the inclusion of `_maps` we implicitly alter the `$navbar-breakpoints` as well. If we remove only from `$navbar-breakpoints` we can keep the `$grid-breakpoints` variable as defined inside the configuration file. e.g.

1. Define before the default value
```scss
// @import...
$navbar-breakpoints: map-remove($grid-breakpoints, "xl", "xxl");
@import "../node_modules/bootstrap/scss/maps";
// @import...
```
2. Override (after) the default value
```scss
// @import...
@import "../node_modules/bootstrap/scss/maps";
$navbar-breakpoints: map-remove($navbar-breakpoints, "xl", "xxl");
// @import...
```
Now that I think about it, maybe it would be better to define the `$navbar-breakpoints: $grid-breakpoints !default;` in the main variables configuration file, like I did in the first version. Otherwise I would need to change the added documentation, that could lead to confusion. 

kind regards

Reply:
Yes I'm a bit confused,

I think I didn't explain well my thought on this. 
I mean the change you mention is totally fine. My comment was more about the `// scss-docs*` thing which isn't used in the doc. Or did I miss something ?

Sorry for the confusion and thanks again for the appreciated contribution.

Reply:
@louismaximepiton Ah, ok. :-) I thought you meant the actual variable...not the docs. But I think it would make more sense either: 
1. to move the extra variable into the main configuration file, or
2. to simply keep only the instruction to define the navbar-breakpoints using the grid-breakpoints var (as in my previous example 1) according to bootstraps docs that recommend to add/remove to/from maps before including the `_maps.scss` file. 

kind regards